src/quantum/setup.py

```diff
@@ -17,7 +17,7 @@
 # This version should match the latest entry in HISTORY.rst
 # Also, when updating this, please review the version used by the extension to
 # submit requests, which can be found at './azext_quantum/__init__.py'
-VERSION = '0.21.0'
+VERSION = '1.0.0'
```

Comment:
Can you confirm you really want to release this extension as a stable version because I saw the API version is preview (2022-01-10-preview)? Also azext.isPreview is set to `true` in azext_metadata.json.
If not, please set version to `1.0.0b1`.

Reply:
extension is a stable version. just our service is in preview, and will be in preview for a long time probably

Reply:
That's ok, but please remove `azext.isPreview: true` in azext_metadata.json.

Reply:
Gotcha I see. In that case, I updated version to be b1